rest/models/types.go

```diff
@@ -136,6 +136,22 @@ const (
 	GTE Comparator = "gte"
 )
 
+// NameComparator is the type of comparison to make for the company_name query parameter in Stock Financials.
+type NameComparator string
+
+const (
+	FULL   NameComparator = "full"
+	SEARCH NameComparator = "search"
+)
+
+// TimeFrame is the type of time frame query parameter for stock financials.
+type Timeframe string
+
+const (
+	ANNUAL    Timeframe = "annual"
```

Comment:
these should be camelcase like the rest of our enums. only acronyms should be all caps (like EQ, LTE, etc)

Reply:
We have two const declarations of Quarterly (one in Frequency, and one here in TimeFrame). Is there a convention for dealing with this?

Reply:
yeah, my convention is to just use the existing Frequency enum instead of adding a new Timeframe enum (it seems like they're functionally the same in this case)